### DISCUSSION TITLE
It is showing March 2021 date and time

### DISCUSSION THREAD

**Discussion by TheGod5** [Chia-Network/chia-blockchain#4747] [Support]
**Describe the bug**
When i open this app this is showing March 2021 Date and Time

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

- OS: [Windows 10]
- OS Version/Flavor: [Latest Version]
- CPU: [Intel i5 10400]

**Additional context**

**Comment by MustDhiMust** [comment]
check your computer time and date

**Comment by TheGod5** [comment]
That was right

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
